java/BUILD.bazel

```diff
@@ -0,0 +1,242 @@
+    size = "small",
+)
+
+java_library(
+    name = "org_ray_ray_runtime",
+    srcs = glob([
+        "runtime/src/main/java/**/*.java"
+    ]) + [
+        ":generate_java_gcs_fbs",
+    ],
+    resources = glob(["runtime/src/main/resources/**"]),
+    visibility = ["//visibility:public"],
+    data = ["//:raylet_library_java"],
+    deps = [
+        ":org_ray_ray_api",
+        "@plasma//:org_apache_arrow_arrow_plasma",
+        "@mvnorg_slf4j_slf4j_api//jar",
+        #"@mvncom_google_code_gson_gson//jar",
+        "@mvncom_fasterxml_jackson_core_jackson_core//jar",
+        "@mvnorg_apache_commons_commons_pool2//jar",
+        "@mvnorg_javassist_javassist//jar",
+        "@mvnorg_objenesis_objenesis//jar",
+        "@mvncom_typesafe_config//jar",
+        "@mvnorg_apache_commons_commons_lang3//jar",
+        "@mvnde_ruedigermoeller_fst//jar",
+        "@mvncom_github_davidmoten_flatbuffers_java//jar",
+        "@mvnredis_clients_jedis//jar",
+        "@mvncommons_io_commons_io//jar",
+        "@mvncom_google_guava_guava//jar",
+        "@mvnnet_lingala_zip4j_zip4j//jar",
+        "@mvnorg_slf4j_slf4j_log4j12//jar",
+        "@mvnorg_ini4j_ini4j//jar",
+        "@mvnorg_ow2_asm_asm//jar",
+    ],
+)
+
+checkstyle_test(
+    name = "org_ray_ray_runtime-checkstyle",
+    target = ":org_ray_ray_runtime",
+    size = "small",
+)
+
+java_binary(
+    name = "org_ray_ray_tutorial",
+    srcs = glob(["tutorial/src/main/java/**/*.java"]),
+    resources = glob(["tutorial/src/main/resources/**"]),
+    visibility = ["//visibility:public"],
+    deps = [
+        ":org_ray_ray_api",
+        ":org_ray_ray_runtime",
+        "@mvncom_google_guava_guava//jar",
+    ],
+)
+
+checkstyle_test(
+    name = "org_ray_ray_tutorial-checkstyle",
+    target = ":org_ray_ray_tutorial",
+    size = "small",
+)
+
+java_library(
+    name = "org_ray_ray_test",
+    srcs = glob(["test/src/main/java/**/*.java"]),
+    deps = [
+        ":org_ray_ray_api",
+        ":org_ray_ray_runtime",
+        "@mvnorg_apache_commons_commons_lang3//jar",
+        "@mvnorg_slf4j_slf4j_api//jar",
+        "@plasma//:org_apache_arrow_arrow_plasma",
+        "@mvnorg_testng_testng//jar",
+        "@mvncommons_collections_commons_collections//jar",
+        "@mvncom_google_guava_guava//jar",
+    ],
+)
+
+checkstyle_test(
+    name = "org_ray_ray_test-checkstyle",
+    target = ":org_ray_ray_test",
+    size = "small",
+)
+
+java_library(
+    name = "org_ray_ray_runtime_test",
+    srcs = glob(["runtime/src/test/java/**/*.java"]),
+    deps = [
+        ":org_ray_ray_api",
+        ":org_ray_ray_runtime",
+        "@mvnorg_apache_commons_commons_lang3//jar",
+        "@mvncom_beust_jcommander//jar",
+        "@mvnorg_testng_testng//jar",
+    ],
+)
+
+checkstyle_test(
+    name = "org_ray_ray_runtime_test-checkstyle",
+    target = ":org_ray_ray_runtime_test",
+    size = "small",
+)
+
+exports_files(["testng.xml"])
+
+java_binary(
+    name = "AllTests",
```

Comment:
also here, `all_tests` for consistent naming convention.

Reply:
OK